lua/entities/gmod_wire_expression2/core/custom/wiring.lua

```diff
@@ -114,3 +114,33 @@ e2function wirelink entity:wirelink()
 	end
 	return this
 end
+
+__e2setcost(10)
+
+--- Links <this> to <ent2> if applicable
+e2function number entity:linkTo(entity ent2)
+	if not IsValid(this) then return self:throw("Invalid Entity!", nil) end
+	if not IsValid(ent2) then return self:throw("Invalid Entity!", nil) end
+	if not isOwner(self, this) then return self:throw("You do not own this entity", nil) end
+	if not isOwner(self, ent2) then return self:throw("You do not own this entity", nil) end
+	
+	if ent2.LinkEnt then
+		local bool = ent2:LinkEnt(this)	
+		if bool == true then return 1 end
```

Comment:
this is one of those times where using the ternary operator is easier `return this:LinkEnt(ent2) and 1 or 0`